src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -609,6 +610,57 @@ public void AssemblyResolve_RequestingAssembly()
         }
 
         [Fact]
+        public void AssemblyResolve_IsNotCalledForCoreLibResources()
+        {
+            bool assemblyResolveHandlerCalled = false;
+            AppDomain.CurrentDomain.AssemblyResolve +=
+                (sender, e) =>
+                {
+                    // This implementation violates the contract. AssemblyResolve event handler is supposed to return an assembly
+                    // that matches the requested identity and that is not the case here.
+                    assemblyResolveHandlerCalled = true;
+                    return typeof(AppDomainTests).Assembly;
+                };
+
+            CultureInfo previousUICulture = CultureInfo.CurrentUICulture;
+            CultureInfo.CurrentUICulture = new CultureInfo("de-CH");
```

Comment:
I believe @tarekgh previously told me that this pattern is problematic for UWP, as UWP doesn't have the notion of per-thread culture and thus changing the thread's culture actually ends up changing the process' culture.  You might need to either restrict the platforms on which this test runs, or use RemoteInvoke to move it out-of-proc.